docs/docs/reference/tags.md

```diff
@@ -86,9 +86,8 @@ must appear on the same line as the tag itself.
 
 ## Snippet
 
-The `snippet` tag, also aliased as `code-block`, marks a range of content in a file
-as a snippet. You can use the [snip](#snip) CLI command to generate snippet files from
-these snippets.
+The `snippet` tag marks a range of content in a file as a snippet.
```

Comment:
```suggestion
> 💡 Prior to version 1.0, bluehawk accepted `code-block` as an alias for `snippet`. Version 1.0 removed the `code-block` alias.

The `snippet` tag marks a range of content in a file as a snippet.
```

Could we consider a warning/info admonition to let folks know that we removed the alias in 1.0? Not sure how widespread use might be, but this seems like fair due diligence (which we can remove in the next major release).